README.md

```diff
@@ -12,10 +12,9 @@ Launch a fully-featured virtual TDP Hadoop cluster with a single command _or_ cu
 
 ## Requirements
 
-- Ansible >= 2.9.6 (to execute the playbooks)
+- Python >= 3.6
```

Comment:
Should we also `pip` as a requirement since it's not installed by default on many distros ?

Reply:
No need directly `pip` but the `venv` package is needed yes and it has a `pip` inside when you create the virtual env.